pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -311,6 +333,10 @@ def newpkgcb(pkgId, name, arch):
                     updateinfo_url = urljoin(remote_url, record.location_href)
                     downloader = self.remote.get_downloader(url=updateinfo_url)
                     downloaders.append([downloader.run()])
+                elif record.type in MODULAR_REPODATA:
+                    modules_url = urljoin(self.remote.url, record.location_href)
+                    modulemd_downloader = self.remote.get_downloader(url=modules_url)
+                    modulemd_results = await modulemd_downloader.run()
```

Comment:
I thought we agreed to leave it at the original place. However, let's not move it. We need to decide for other items where it's the best to await for them. All good.